static/app/utils/number/formatMetricUsingUnit.tsx

```diff
@@ -161,7 +110,7 @@ const METRIC_UNIT_TO_SHORT: Record<FormattingSupportedMetricUnit, string> = {
   exabyte: 'EB',
   exabytes: 'EB',
   none: '',
-};
+} as const satisfies Record<string, string>;
```

Comment:
This seems slightly backwards to me? The original code was enforcing that every supported metric unit had a corresponding short unit, but now the types are derived from the units so the object becomes the list of "supported metric units." Might be a little unexpected.

Would it make more sense to convert the `as const` array to a string literal type and then maintain `const METRIC_UNIT_TO_SHORT: Record<FormattingSupportedMetricUnit, string>`?

Reply:
decided to re-name it here: a03583a2